index.html

```diff
@@ -709,6 +709,78 @@ <h2>Exposing MediaStreamTrack source background blur support</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Exposing MediaStreamTrack source automatic face framing support</h2>
+    <p>Some platforms or User Agents may provide built-in support for automatic
+       continuous framing based on the position of human faces within the field
+       of view, in particular for camera video streams.
+       Web applications may either want to control or at least be aware that
+       automatic continuous human face framing is applied at the source level.
+       This may for instance allow the web application to update its UI or to
+       not apply human face framing on its own.
+       For that reason, we extend {{MediaStreamTrack}} with the following
+       properties.
+    </p>
+    <p>The WebIDL changes are the following:</p>
+    <pre class="idl"
+>partial dictionary MediaTrackSupportedConstraints {
+  boolean faceFraming = true;
+};
+
+partial dictionary MediaTrackCapabilities {
+  sequence&lt;boolean&gt; faceFraming;
+};
+
+partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean faceFraming;
+};
+
+partial dictionary MediaTrackSettings {
+  boolean faceFraming;
+};</pre>
+    <div class="note">
+      If a need for automatic single-shot framing arises,
+      {{MediaTrackConstraintSet/faceFraming}} booleans could be replaced with
+      faceFramingMode {{MeteringMode}} {{DOMString}}s.
```

Comment:
Do we need this note, it seems we could file an issue instead?

Reply:
Ok, I'll do that.